src/org/labkey/test/tests/LinkedSchemaTest.java

```diff
@@ -424,7 +424,7 @@ void deleteLinkedSchema()
     public void lookupTest()
     {
         String sourceContainerPath = "/" + getProjectName() + "/" + SOURCE_FOLDER;
-        _schemaHelper.createLinkedSchema(getProjectName(), TARGET_FOLDER, "BasicLinkedSchema", sourceContainerPath, null, "lists", "NIMHDemographics,NIMHPortions", null);
+        _schemaHelper.createLinkedSchema(getProjectName() + "/" + TARGET_FOLDER, "BasicLinkedSchema", sourceContainerPath, null, "lists", "NIMHDemographics,NIMHPortions", null);
```

Comment:
At some point it might be helpful to have a paths or urls util do the work of assembling strings of <container name><delimiter char> etc.   Not asking that we do that now, but we should give some thought to how to approach it